
console.log("hello");
console.log("hello");
try{
    console.log(a);
}
catch(error)
{
    console.log("caught a error a is not defined");
    console.log(error);
}

console.log("hello2");
console.log("hello2");




let b=15;
console.log("hello");
console.log("hello");
try{
    console.log(b);
}
catch(error)
{
    console.log("caught b error a is not defined")
    console.log(error);
}

console.log("hello2");
console.log("hello2");
